tests/bin/ds-container-create.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -ex
+
+# https://fy.blackhats.net.au/blog/html/2020/03/28/389ds_in_containers.html
+
+docker run \
+    --detach \
+    --name=${NAME} \
+    --hostname=${HOSTNAME} \
+    -e DS_DM_PASSWORD=${PASSWORD} \
+    quay.io/389ds/dirsrv
+
+sleep 25
```

Comment:
Can we replace this hardcoded 25s wait with a polled status check of some kind?

Reply:
+1